### ISSUE TITLE
not woking with @storybook/react 4.1.4

### ISSUE THREAD

**Issue by SokolovAlex** [react-theming/storybook-addon-material-ui#90]
create simple storybook project from 
https://storybook.js.org/basics/guide-react/

/// package.json
{
  "scripts": {
    "storybook": "start-storybook -p 9001 -c .storybook"
  },
  "devDependencies": {
    "@storybook/react": "^4.1.4"
  },
  "dependencies": {
    "@babel/core": "^7.2.2",
    "babel-loader": "^8.0.4",
    "react": "^16.7.0",
    "react-dom": "^16.7.0",
    "storybook-addon-material-ui": "^0.8.11"
  }
}

/// .storybook/config.js
import { configure } from '@storybook/react';

function loadStories() {
  require('../stories/index.js');
  // You can require as many stories as you need.
}
configure(loadStories, module);

add addon
import 'storybook-addon-material-ui/register';

after that storybook cant starting
![image](https://user-images.githubusercontent.com/5417142/50475407-b4f2b600-09d5-11e9-873d-dff92104111a.png)

what did i do wrong?

**Comment by UsulPro** [comment]
Hi
You need to use the `"storybook-addon-material-ui@0.9.0-alpha.16`. It's the latest version

**Comment by SokolovAlex** [comment]
I try both
"storybook-addon-material-ui": "^0.9.0-alpha.16"
and 
"storybook-addon-material-ui": "^0.8.11"

same result

**Comment by UsulPro** [comment]
@SokolovAlex Ok, I'll check it, thanks for reporting this

**Comment by alistair-hmh** [comment]
It's hanging for me also...

```json
    "@storybook/addon-actions": "^4.0.9",
    "@storybook/addon-links": "^4.0.9",
    "@storybook/addons": "^4.0.9",
    "@storybook/react": "^4.0.9",
    "@storybook/addon-a11y": "^4.0.9",
    "@storybook/addon-info": "^4.0.9",
    "@storybook/addon-knobs": "^4.0.9",
    "@storybook/addon-storysource": "^4.0.9",
    "storybook-addon-material-ui": "0.9.0-alpha.16",
```

**Comment by alistair-hmh** [comment]
@UsulPro - just out of interest? Why does material ui register break with the new Storybook React?

**Comment by UsulPro** [comment]
Hi @alistair-hmh 
I didn't dig but looks like it transpiles into something incompatible. Possible Babel updating would help. Here is a working example https://github.com/storybooks/addon-development-kit/tree/version-1

**Comment by SKCrawford** [comment]
Has anyone progress been made towards resolving this issue? I cannot downgrade storybook because of create-react-app. Otherwise, is there any other (simple) way to provide the theme manually?

**Comment by SleepWalker** [comment]
This may be due to add-on, because starting from 4.1.x you can not import any core node modules in addons. This will lead to compile error, but due to https://github.com/storybooks/storybook/issues/5584 it will hang without showing error.
You can apply patch manually simply to see an actual error, or try to test with storybook 5, where the bug with hidden errors is fixed

**Comment by UsulPro** [comment]
Solved in #92
